virtcontainers/clh.go

```diff
@@ -294,13 +246,18 @@ func (clh *cloudHypervisor) startSandbox(timeout int) error {
 	if err != nil {
 		return fmt.Errorf("failed to launch cloud-hypervisor: %s, error messages from log: %s", err, strErr)
 	}
+	clh.state.PID = pid
```

Comment:
@jcvenegas Can you share a snapshot of jaeger trace output when launching?  I'm curious how long it takes to start the VMM itself, and if it makes sense to do this at top of function instead of here (similar to what's in FC)?